null_jobs_backend/settings.py

```diff
@@ -241,3 +242,53 @@
 
 # GOOGLE AUTH
 SITE_ID = 2
+
+LOG_DIR = os.path.join(BASE_DIR, "logs")
+
+if not os.path.exists(LOG_DIR):
+    os.makedirs(LOG_DIR)
+
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "custom_format": {
+            "format": "%(process)d - %(asctime)s - %(levelname)s - %(message)s\n%(stack_info)s",
+            "datefmt": "%D-%m-%Y %H:%M:%S %P",  # Includes timezone and date format
+        },
+    },
+    "handlers": {
```

Comment:
Have you discussed with anyone about where to store/print the logs, for example StreamHandler or FileHandler? Because here you are only using the FileHandler but what if we need some minimal logs on the console as well, such as INFO messages?